pysparkling/sql/ast/ast_to_python.py

```diff
@@ -0,0 +1,590 @@
+import ast
+import logging
+
+from sqlparser import string_to_ast
```

Comment:
This is very nice!

Reply:
Thanks :)

It's quite powerful as it already converts any SQL statement into a syntax tree, it should allow supporting `spark.sql(statement)` but there is a lot of mapping to do first :)